docs/docs/glossary.md

```diff
@@ -0,0 +1,70 @@
+
+A **job** brings one process graph to the back-end and organizes its execution, which may or may not induce costs. Jobs furthermore allow to run process graphs from different _data views_ (see section on [data views](views.md)). Views define at which resolution and extent we look at the data during processing and hence allow to try out process graphs on small subsets, or work interactively within web map applications. For more information about jobs and their evaluation types, see the section on [jobs](jobs.md).
+
+In this context OpenEO will:
+
+1. consider, or allow to consider, band as a dimension
+2. consider `collections` to consist of one _or more_ collections, as argument to all functions; allow filtering on a particular collection, or joining them into a single collection
+3. allow filtering on attributes, e.g. on cloud-free pixels, or pixels inside a `MULTIPOLYGON` describing the floodplains of the Danube. This filters on attributes rather than dimensions.
+4. Provide generic aggregate operations that aggregate over one or more dimenions. Clients may provide dimension-specific aggregation functions for particular cases (such as `min_time`) 
+
+### Aggregation vs. resampling
+
+***Aggregation*** computes new values from sets of values that are _uniquely_ assigned to groups. It involves a grouping predicate (e.g. monthly, 100 m x 100 m grid cells; think of SQL's `group_by`), and an aggregation function (e.g., `mean`) that computes one or more new values from the original ones.
+
+Examples:
+
+- a time series aggregation may return a regression slope and intercept for every pixel time series, for a single band (group by: full time extent)
+- a time series may be aggregated to monthly values by computing the mean for all values in a month (group by: months)
+- _spatial_ aggregation involves computing e.g. _mean_ pixel values on a 100 x 100 m grid, from 10 m x 10 m pixels, where each original pixel is assigned uniquely to a larger pixel (group by: 100 m x 100 m grid cells)
+
+Note that for the first example, the aggregation function not only requires time series values, but also their time stamps.
+
+***Resampling*** is a broader term where we have data at one resolution, and need values at another (also called _scaling_). In case we have values at a 100 m x 100 m grid and need values at a 10 m x 10 m grid, the original values will be reused many times, and may be be simply assigned to the nearest high resolution grid cells ("nearest neighbor"), or may be interpolated somehow (e.g. by bilinear interpolation). Resampling from finer to coarser grid by nearest neighbor may again be a special case of aggregation.
```

Comment:
Resampling from finer to coarser grid by techniques other than "nearest neighbor" may be a special case of aggregation too, may it not?

Reply:
Moved to issue #26 for later handling.